pkg/geo/geogfn/topology_operations.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package geogfn
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/geo"
+	"github.com/cockroachdb/errors"
+	"github.com/golang/geo/r3"
+	"github.com/golang/geo/s2"
+	"github.com/twpayne/go-geom"
+)
+
+// Centroid returns the Centroid of a given Geography.
+//
+// Note: In the case of (Multi)Polygon Centroid result, it doesn't mirror with
+// `PostGIS`'s result. Though we are using the same algorithm that is divide
```

Comment:
replace from `Though we are`... with:

We are using the same algorithm of dividing into triangles. However, The PostGIS implementations differs as it cuts triangles in a different way - namely, it is fixes the first point in the exterior ring as the first point of the triangle whereas this implementation moves the reference point to be each ring. In our implementation, we always update the reference point to be the first point of the ring when moving from one ring to another.

Reply:
done!